setup.py

```diff
@@ -3,7 +3,7 @@
 
 setuptools.setup(
     name='mkdocs-monorepo-plugin',
-    version='0.5.3',
+    version='0.5.4',
```

Comment:
A note in `CHANGELOG.md` for this change would be good too!  Something like `Fixed edit URLs for included pages.`